sdk/core/core-http/src/policies/deserializationPolicy.ts

```diff
@@ -75,10 +82,12 @@ export const DefaultDeserializationOptions: DeserializationOptions = {
 export class DeserializationPolicy extends BaseRequestPolicy {
   public readonly jsonContentTypes: string[];
   public readonly xmlContentTypes: string[];
+  public readonly xmlCharKey: string;
 
   constructor(
     nextPolicy: RequestPolicy,
     deserializationContentTypes: DeserializationContentTypes | undefined,
+    parsingOptions: { xmlCharKey?: string } | undefined,
```

Comment:
rather than putting a literal `{ xmlCharKey?: string }` everywhere, can we write this down as a type? It'll make it much easier to add additional parsingOptions later if needed.